src/utils/isCurator.ts

```diff
@@ -0,0 +1,17 @@
+import { activeAccount } from '../stores';
+import { get } from 'svelte/store';
+import type { Bounty } from '../types/bounty';
+import type { AccountInfo } from '../types/account';
+import { getRelevantMultisig } from './getRelevantMultisig';
+import type { ChildBounty } from '../types/child-bounty';
+
+/**
+ * Returns whether the given account is the curator of the given bounty or child bounty
+ * directly or through a multisig.
+ */
+export function isCurator(bounty: Bounty | ChildBounty, account?: AccountInfo): boolean {
+	account = account || get(activeAccount);
```

Comment:
```suggestion
export function isCurator(bounty: Bounty | ChildBounty): boolean {
	const account = get(activeAccount);
```
The second parameter is not used anywhere

Reply:
I wanted to make the function more generic, but we can add that if needed in the future.